scripts/deploymentConfig.js

```diff
@@ -0,0 +1,12 @@
+require("dotenv").config({ path: "./.env" })
+
+module.exports = {
+	privateKey: process.env.PRIVATE_KEY,
+	mainnetRpcUrl: "https://mainnet.veblocks.net/",
+	testnetRpcUrl: "https://testnet.veblocks.net/",
+	tokenBeneficiaryAddress: "0x57e977Ff64FDD6b352FE0adA9D7a2f759F2cAb4a",
```

Comment:
Should we just make this the deployer? Guess we can always update the config so not a big deal

Reply:
okay, then the deployer will allocate tokens accordingly